handbook/engineering/core-application/index.md

```diff
@@ -4,49 +4,43 @@
 
 While you could think this is an angry cloud, it's actually a fierce and determined one 😃.
 
-The core application team owns a number of [fundamental areas](#areas-of-ownership) in our product and code base.
+## Areas of ownership
+The Core Application team has broad [area ownership](areas-of-ownership.md) within the Sourcegraph product delivery organization. We are responsible for [code host connection and repository management](areas-of-ownership.md#code-host-connections-and-repositories-management), [administration experience](areas-of-ownership.md#administration-experience), [licensing, finance and subscription management](areas-of-ownership.md#licensing-and-subscription-management), [authorization, authentication, and IAM](areas-of-ownership.md#identity-and-access-management). The team also owns other [platform-level developer enablement themes](areas-of-ownership.md#developer-enablement).
 
-Our current work-streams are:
+## Members 
 
-1. **Core application**: Building, securing and scaling our multi-tenant hosted version of Sourcegraph for customers that do not want to deploy Sourcegraph on-premise. Also meeting the needs of larger and larger enterprises to support those deals (e.g. Perforce support)
-2. **Backend platform**: Make it easy for teammates of different experience levels to onboard and contribute to backend code. Hide and remove non essential complexity from engineers working on new product features and use cases. Promote and champion consistency and simplicity across all backend code by building shared tools, libraries and infrastructure for common use cases. Scale and maintain said infrastructure. Create leverage for and accelerate other teams.
+<!-- Due to the markdown renderer that we use, the indentation here is sensitive. If you want to change the indentation, check that it renders correctly locally with `make serve` -->
 
-## Areas of ownership
+- [Ryan Phillips](../../company/team/index.md#ryan-phillips-he-him) ([Product Manager](../../product/roles/index.md#product-manager))
+- [Quinn Keast](../../company/team/index.md#quinn-keast-he-him) ([Product Designer](../../product/roles/index.md#product-designer))
+- [Rafal Leszczynski](../../company/team/index.md#rafal-leszczynski-he-him) ([Engineering Manager](../roles.md#engineering-manager)) {#core-application-eng}
+  - [Joe Chen](../../company/team/index.md#joe-chen)
+  - [Ryan Slade](../../company/team/index.md#ryan-slade-he-him)
+  - [Alan Harris](../../company/team/index.md#alan-harris-he-him)
+  - [Artem Ruts](../../company/team/index.md#artem-ruts-he-him)
+  - [Asdine El Hrychy](../../company/team/index.md#asdine-el-hrychy)
+  - [Indradhanush Gupta](../../company/team/index.md#indradhanush-gupta-he-him)
+  - Milan Freml (Aug 9)
+  - Rafal (Oct 11)
```

Comment:
Sidenote: I've seen initials being used for teammates yet to join, potentially to protect them from unseen circumstances.

Reply:
👆